src/wp-content/themes/twentytwelve/functions.php

```diff
@@ -203,7 +185,8 @@ function twentytwelve_scripts_styles() {
 
 	$font_url = twentytwelve_get_font_url();
 	if ( ! empty( $font_url ) ) {
-		wp_enqueue_style( 'twentytwelve-fonts', esc_url_raw( $font_url ), array(), null );
+		$font_version = ( 0 === strpos( (string) twentytwelve_get_font_url(), get_template_directory_uri() . '/' ) ) ? '20230328' : null;
```

Comment:
Can we use `str_starts_with` here?

Reply:
Thanks! I thought of that, too, but no. These themes should work with older versions of both PHP and WordPress (without the [polyfill added in 5.9](https://core.trac.wordpress.org/changeset/52040)). That might have been easier to write :)